js/utils/synthutils.js

```diff
@@ -216,7 +216,7 @@ const SAMPLECENTERNO = {
     'celeste': ['C3', 27],  // pitchToNumber('C', 3, 'C Major')],
     'vibraphone': ['C5', 51],
     'xylophone': ['C4', 39],
-    'viola': ['C4', 51]
+    'viola': ['D4', 55]
```

Comment:
shouldn't this be D4 52 ???

Reply:
Whatever it is suppose to be, it results in the correct sound.

Probably it is suppose to be `53` as `52` is only a half step from `51`.